intro-HPC/ch_quick_reference_guide.tex

```diff
@@ -1,52 +1,54 @@
 \chapter{\hpc Quick Reference Guide}
 \label{ch:quick-reference-guide}
 
+Remember to subsitute the usernames, login nodes, file names, \ldots for your own.
+
 \begin{tabular}{|l|l|} \hline
 \multicolumn{2}{|c|}{\strong{Login}} \\ \hline
-Login             & ssh <{}vsc-account>{}@\loginnode \\ \hline
-Where am I?       & hostname \\ \hline
-Copy to \hpc      & scp foo.txt <{}vsc-account>{}@\loginnode: \\ \hline
-Copy from \hpc    & scp <{}vsc-account>{}@\loginnode:foo.txt . \\ \hline
-Setup ftp session & sftp <{}vsc-account>{}@\loginnode \\ \hline
+Login             & \texttt{ssh \userid{}@\loginnode{}}\\ \hline
+Where am I?       & \texttt{hostname} \\ \hline
+Copy to \hpc      & \texttt{scp foo.txt \userid{}@\loginnode{}:} \\ \hline
+Copy from \hpc    & \texttt{scp \userid{}@\loginnode{}:foo.txt .} \\ \hline
+Setup ftp session & \texttt{sftp \userid{}@\loginnode{}} \\ \hline
 \end{tabular}
 
 \begin{tabular}{|l|l|} \hline
 \multicolumn{2}{|c|}{\strong{Modules}} \\ \hline
-List all available modules & module av \\ \hline
-List loaded modules        & module list \\ \hline
-Load module                & module load <{}name>{} \\ \hline
-Unload module              & module unload <{}name>{} \\ \hline
-Unload all modules         & module purge \\ \hline
-Help on use of module      & module help \\ \hline
+List all available modules & \texttt{module av} \\ \hline
+List loaded modules        & \texttt{module list} \\ \hline
+Load module                & \texttt{module load <{}name>{}} \\ \hline
+Unload module              & \texttt{module unload <{}name>{}} \\ \hline
```

Comment:
use `example` rather than `<name>` (which confuses people, they think the `<` and `>` must be included...)